code/app/src/main/java/com/example/sigma_blue/entity/item/Item.java

```diff
@@ -342,13 +344,13 @@ public List<Tag> getTags() {
     }
 
     /**
-     * Returns the list of tag names
+     * Returns the list of document tag ids.
      *
-     * @return
+     * @return a list of the document tag ID
      */
-    public ArrayList<String> getTagNames() {
-        return new ArrayList<>(tags.stream().map(Tag::getDocID).collect(Collectors
-                .toList()));
+    public ArrayList<String> getTagDocIDs() {
+        return tags.stream().map(Tag::getDocID).collect(Collectors
+                .toCollection(ArrayList::new));
```

Comment:
faster construction because this version no longer construct twice.

Reply:
Assuming this is functionally the same I have no complaints.